cumulusci/tasks/bulkdata/delete.py

```diff
@@ -53,9 +53,11 @@ def _init_options(self, kwargs):
             raise TaskOptionsError(
                 "Criteria cannot be specified if more than one object is specified."
             )
-        self.options["hardDelete"] = process_bool_arg(self.options.get("hardDelete"))
+        self.options["hardDelete"] = process_bool_arg(
+            self.options.get("hardDelete", False)
+        )
         self.options["ignore_row_errors"] = process_bool_arg(
-            self.options.get("ignore_row_errors")
+            self.options.get("ignore_row_errors", False)
         )
```

Comment:
Better to use `.get("option_name") or False` for all of these. The option could be explicitly set to null in yaml.